### PR TITLE
actions: Add actions for protobuf linter

### PR DESCRIPTION
**Pull request by iamniting**
for reference
https://github.com/marketplace/actions/protolint

Signed-off-by: Nitin Goyal <nigoyal@redhat.com>

### CODE REVIEW

File: .github/workflows/protolint.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    branches: [ '*' ]
+  pull_request:
+    branches: [ '*' ]
+
+jobs:
+  test-lint:
+    runs-on: ubuntu-latest
+    steps:
+    - name: checkout source
+      uses: actions/checkout@v1
```
Comment (Madhu-1) [inline .github/workflows/protolint.yaml:14]: use v2 here?

File: extras/.protolint.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# visit below link for info
+# https://github.com/yoheimuta/protolint/blob/master/_example/config/.protolint.yaml
```
Comment (Madhu-1) [inline extras/.protolint.yaml:2]:
```suggestion
Reference https://github.com/yoheimuta/protolint/blob/master/_example/config/.protolint.yaml
```

File: .github/workflows/protolint.yaml (hunk 1 above)
Comment (iamniting) [reply .github/workflows/protolint.yaml]: DOne

File: extras/.protolint.yaml (hunk 2 above)
Comment (iamniting) [reply extras/.protolint.yaml]: Done

File: .github/workflows/protolint.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+name: protolint
+
+on:
+  push:
```
Comment (Madhu-1) [inline .github/workflows/protolint.yaml:4]: For me it looks like we don't need to run actions `when code is pushed to any branch in this repository` push can we add the push later? if required.

**Review by Madhu-1 — APPROVED** [review]

**Comment by Madhu-1** [comment]
Merging this one to check protolint is working on other PR's

**Pull request merged**